adapters/appnexus.go

```diff
@@ -68,7 +68,8 @@ type appnexusImpExt struct {
 }
 
 func (a *AppNexusAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pbs.PBSBidder) (pbs.PBSBidSlice, error) {
-	anReq := makeOpenRTBGeneric(req, bidder, a.FamilyName())
+	mediaTypes := []pbs.MediaType{pbs.MEDIA_TYPE_BANNER, pbs.MEDIA_TYPE_VIDEO}
```

Comment:
might make sense to rename this as "supported media types" to be clear that this isn't the media types requested by the buyer, and to move it into the adapter definition - in other words `a.SupportedMediaTypes()`